lib/improver/tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -321,9 +339,8 @@ def test_basic(self):
         threshold = AuxCoord(0.03, units='mm hr-1')
         result = plugin.construct_extract_constraint(diagnostic,
                                                      threshold)
-        expected = ("Constraint(name='probability_of_rainfall_rate',"
-                    " coord_values={'threshold': 0.03})")
-        self.assertIsInstance(result, iris.Constraint)
+        expected = ("iris.Constraint(name='probability_of_rainfall_rate', "
+                    "threshold=lambda cell: 0.03*0.99 < cell < 0.03*1.01)")
         self.assertEqual(str(result), expected)
```

Comment:
Is this function not returning a String. Need to check that a string is returned
`self.assertIsInstance(result, str)`
and remove str from self.assertEqual

Reply:
:white_check_mark: